pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -1,11 +1,19 @@
 ---
+<%
+  hoc = DCDO.get('soon-hoc', false) || DCDO.get('actual-hoc', false)
+  share_image = hoc ? 'celeb-challenge.jpg' : 'default-og-image.jpg'
+%>
 theme: responsive_wide
 banner: null
 layout: wide
 video_player: true
 social:
   "og:title": "<%= I18n.t(:og_title) %>"
   "og:description": "<%= I18n.t(:og_description)%>"
+  "og:image" : "<%= 'https://' + request.host + '/images/fit-1220/social-media/' + share_image %>"
+  "og:image:width": '1220'
+  "og:image:height": '640'
```

Comment:
I used 1220x640 to match the defaults, not sure if that's right.

https://github.com/code-dot-org/code-dot-org/blob/f6a7749bdbdbb68416052cb2d8a485af9c22a3dd/pegasus/router.rb#L490-L492